### PR TITLE
Style fixes for VBC

### PR DESCRIPTION
**Pull request by Anush2303**
## Previous Behavior

Styling issues in VBC and LineChart as compared to their V8 version.

## New Behavior

Fixed some styling issues , VBC and LC in V9 looking almost same as V8.

## Related Issue(s)

- Fixes #

### CODE REVIEW

File: packages/react-components/react-charts-preview/stories/VerticalBarChart/VerticalBarChartDefault.stories.tsx [hunk 1]
```diff
@@ -1,9 +1,10 @@
 import * as React from 'react';
-import { VerticalBarChart, IVerticalBarChartProps, IVerticalBarChartDataPoint } from '../../src/VerticalBarChart';
+import { VerticalBarChart, IVerticalBarChartProps, IVerticalBarChartDataPoint, ILineChartLineOptions} from '../../src/VerticalBarChart';
 import { DefaultPalette } from '@fluentui/react/lib/Styling';
 import { ChoiceGroup, IChoiceGroupOption } from '@fluentui/react/lib/ChoiceGroup';
 import { Checkbox } from '@fluentui/react/lib/Checkbox';
 import { Switch } from '@fluentui/react-components';
+import { IRenderFunction } from '@fluentui/react/lib/Utilities';
```
Comment (AtishayMsft) [inline packages/react-components/react-charts-preview/stories/VerticalBarChart/VerticalBarChartDefault.stories.tsx:7]: don't use v8 functions

Comment (Anush2303) [reply packages/react-components/react-charts-preview/stories/VerticalBarChart/VerticalBarChartDefault.stories.tsx:7]: Okay. will remove this

Comment (Anush2303) [reply packages/react-components/react-charts-preview/stories/VerticalBarChart/VerticalBarChartDefault.stories.tsx:7]: > don't use v8 functions

So is there any alternate function for this?

**Review by AtishayMsft — APPROVED** [review]

File: packages/react-components/react-charts-preview/stories/VerticalBarChart/VerticalBarChartDefault.stories.tsx (hunk 1 above)
Comment (AtishayMsft) [reply packages/react-components/react-charts-preview/stories/VerticalBarChart/VerticalBarChartDefault.stories.tsx:7]: To be fixed separately in another PR.

**Pull request merged**